### DISCUSSION TITLE
Inline Styling

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#20795] [Help] [answered]
Hi. Is inline styling possible with Next.js or would I have to do something like `className={styles.style}`?

**Comment by jamesmosier** [comment, marked as answer]
What do you mean by "inline styling"? 

One option would be to use [CSS modules](https://nextjs.org/docs/basic-features/built-in-css-support#adding-component-level-css) as you mentioned.

Or you could use [inline styles](https://nextjs.org/docs/basic-features/built-in-css-support#css-in-js) on each element: `<p style={{ color: 'red' }}>hi there</p>`

or `styled-jsx`:

```js
function HelloWorld() {
  return (
    <div>
      Hello world
      <p>scoped!</p>
      <style jsx>{`
        p {
          color: blue;
        }
        div {
          background: red;
        }
        @media (max-width: 600px) {
          div {
            background: blue;
          }
        }
      `}</style>
      <style global jsx>{`
        body {
          background: black;
        }
      `}</style>
    </div>
  )
}
```

or any of the other CSS-in-JS solutions (i.e. styled-components, emotion, etc.)

**Reply by ghost** [reply]
I meant like the `<a style="...">words</a>` inline styling. Thanks.
